tests/test_multidict.py

```diff
@@ -224,6 +224,23 @@ def test__iter__(
         d = cls([("key", "one"), ("key2", "two"), ("key", 3)])
         assert list(d) == ["key", "key2", "key"]
 
+    def test__contains(
+        self,
+        cls: Union[
+            Type[MultiDict[Union[str, int]]],
+            Type[CIMultiDict[Union[str, int]]],
+        ],
+    ) -> None:
+        d = cls([("key", "one"), ("key2", "two"), ("key", 3)])
+
+        assert list(d) == ["key", "key2", "key"]
+
+        assert "key" in d
+        assert "key2" in d
+
+        assert "foo" not in d
+        assert 42 not in d
```

Comment:
Can we also try the opposite code path?
```suggestion
        assert 42 not in d
        assert not (42 in d)
```
(this will probably need a noqa comment)

Reply:
If you want, they are fundamentally the same though (i.e. it won't test anything different in our code):
https://docs.python.org/3/reference/expressions.html#not-in

Reply:
Ah, right. I didn't realize there's no separate dunder method for that. Marking as resolved.